src/java/org/apache/cassandra/db/RangeTombstoneList.java

```diff
@@ -146,11 +153,16 @@ public void add(RangeTombstone tombstone)
      * This method will be faster if the new tombstone sort after all the currently existing ones (this is a common use case),
      * but it doesn't assume it.
      */
-    public void add(ClusteringBound<?> start, ClusteringBound<?> end, long markedAt, int delTime)
+    public void add(ClusteringBound<?> start, ClusteringBound<?> end, long markedAt, long delTime)
```

Comment:
Now that we have the uint version of this method, this old version is unused and can be removed.

Reply:
Yeah but it's always been public and I don't want to burden any future callers with the uint conversion burden. I would prefer to leave it?

Reply:
I think the version taking `int delTimeUnsignedInteger` should be `private`. The version taking `long delTime` is dead code without any test coverage, so it should be deleted. AFAIK `RangeTombstoneList` is not a public API so there is no reason to keep old unused methods.

Reply:
Will do.